modules/apps/29-fee/keeper/grpc_query.go

```diff
@@ -17,12 +17,11 @@ var _ types.QueryServer = Keeper{}
 
 // IncentivizedPackets implements the Query/IncentivizedPackets gRPC method
 func (k Keeper) IncentivizedPackets(goCtx context.Context, req *types.QueryIncentivizedPacketsRequest) (*types.QueryIncentivizedPacketsResponse, error) {
+	ctx := sdk.UnwrapSDKContext(goCtx).WithBlockHeight(int64(req.QueryHeight))
```

Comment:
I don't think it makes sense to move this context unwrapping to above the `nil` check. This uses `WithBlockHeight()` and passes the `req.QueryHeight` as an arg. 

As you can see the linter is complaining with - SA5011: possible nil pointer dereference (staticcheck)

Reply:
Would you recommend we leave the context wrapping always after the req == nil check or separate the two pieces of logic here (wrapping the context, setting the query height)

Reply:
I think either is fine, but technically the fail fast approach if `req == nil` is probably more efficient. No need to unwrap the context if the request is empty because we'll fail anyways.